defaults/main.yml

```diff
@@ -31,15 +31,14 @@ extra_environment_variables: {}
 # Docker Image Versions
 portal_image_version: latest
 hub_image_version: latest
-reverse_proxy_image_version: 1.2.1
+reverse_proxy_image_version: 2.2
```

Comment:
need to update docs that this no longer refers to the nginx version but now traefik